.github/workflows/production-workflow.yml

```diff
@@ -0,0 +1,60 @@
+name: Deploy to Production
+
+on:
+  push:
+    branches:
+      - production
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Commit
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: .nvmrc
+          cache: npm
+
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: Run Tests
+        run: npm test
+
+      - name: Prepare Wrangler Config
+        run: |
+          SENTRY_DSN=$(printf '%s\n' "${{ secrets.SENTRY_DSN }}" | sed -e 's/[\/&]/\\&/g')
```

Comment:
I think this mostly comes down to personal preference, I've always used `printf` for this kind of thing. Some part of my brain also says that are some edge cases where it actually proves to be more reliable/consistent (I think around escapes etc.) but I'm not of the specifics without doing some deep digging

Reply:
Fair enough. I believe `printf` is technically more portable because `echo` used to not have options, and handle things like `\n` differently (although I doubt we're using a system where that's the case). I don't foresee any problem using `echo` here (famous last words 👀), and feel that is simpler for what's being done here, but equally, `printf` remains fine here, so I'm not too fussed either way :-)